README.md

```diff
@@ -1 +1,289 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse Project Two] <a name="about-project"></a>
+
+> This is my first project from microverse school where i have been assigned to check my knowledge in the use of linters.
+
+**[Hello Microverse Project Two]** is a project on use of linters that only focuses on two languages which are html/css at the moment but could be modified later.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Added index.html]**
+- **[Added styles.css]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Web browser
+
+2. Code editor(visual studio code)
+
+3. Git/Github(version control system)
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1.git clone <the ssh url of the given repository>
+
+2.cd <the name of the repository>
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+To run the project on local machine execute the following command
+
+1.code .(to open on visual studio code)
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+1.Open index.html file on your visual studio then use live server to open it on your prefered browser.
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Brenda Lutiku
+
+👤 **Author1**
+
+- GitHub: [Github]
+- Twitter: [Github]
+- LinkedIn: [LinkedIn]
+
+👤 **Author2**
+
+- GitHub: [Brenda Lutiku)
+- Twitter: [Brenda Lutiku)
+- LinkedIn: [Brenda Lutiku)
+
```

Comment:
- [ OPTIONAL] This is a solo project, I suggest that you kindly delete the second author. Please also add your author details according to the clues provided. add your GitHub, Twitter, and LinkedIn. You can create these accounts if you don't have them yet